### DISCUSSION TITLE
Multiple Inputs in Neural Prophet

### DISCUSSION THREAD

**Discussion by karthikraja95** [ourownstory/neural_prophet#220] [Q&A - get help using NeuralProphet]
How can I use multiple inputs in Neural Prophet? By looking at the example notebooks I see we need to have two columns ds as date and y as a target variable?

My question is how to add other features with ds (Date). For example, some categorical features to be considered by the neural prophet model?

**Comment by HansikaPH** [comment]
For categorical features, please see the discussion #139

**Reply by karthikraja95** [reply]
@HansikaPH Thanks

**Comment by ourownstory** [comment]
@karthikraja95 

> By looking at the example notebooks I see we need to have two columns ds as date and y as a target variable?

Yes, that is the minimal input.

> My question is how to add other features with ds (Date). 

May [events](https://github.com/ourownstory/neural_prophet/blob/master/example_notebooks/events_holidays_peyton_manning.ipynb) or a type of [regressor](https://ourownstory.github.io/neural_prophet/model/future-regressors/) serve your needs?

**Reply by karthikraja95** [reply]
@ourownstory I have multiple features (both categorical and non-categorical) that I would like the model to consider as inputs along with ds to predict the target variable y? Is this possible with the neural prophet?

**Reply by ourownstory** [reply]
@karthikraja95 yes, you can model numeric features with [regressors](http://neuralprophet.com/model/lagged-regressors/) and you can convert categorical features to one binary feature per category and then model them as regressors or events.
